inspire/dojson/institutions/schemas/insitutions-0.0.1.json

```diff
@@ -129,11 +168,6 @@
             "type": "array",
             "uniqueItems": true
         },
-        "non_public_note": {
-            "description": "FIXME: what is the difference with hidden_note!?",
-            "title": "Non public note",
-            "type": "string"
-        },
```

Comment:
Yeah, I noticed. This takes it from the different MARC field. Apparently no jobs have 595 fields.